apps/src/applab/designElements/EnumPropertyRow.jsx

```diff
@@ -20,23 +21,26 @@ export default class EnumPropertyRow extends React.Component {
   };
 
   render() {
-    let options = this.props.options.map(function(option, index) {
+    const {options, displayOptions = [], desc} = this.props;
+    const {selectedValue} = this.state;
```

Comment:
Note to self: Perhaps I'm just not familiar with this syntax or the reasoning for declaring these values here, but this seems like unnecessary complication.

Reply:
This is a modern stylistic cleanup. Another project that I work on requires this style (barring use of `this.props.foo` everywhere in the class). I've really grown to like it.

It also helps with ensuring the default values are in one place (assuming you are relying on `defaultProps`)

See: https://dev.to/arnas/destructuring-react-props-for-the-cleaner-code-293

Reply:
👍